src/main/java/edu/hawaii/its/api/controller/GroupingsRestController.java

```diff
@@ -100,14 +100,22 @@ public ResponseEntity<String> hello() {
         return ResponseEntity.ok("University of Hawaii UHGroupings");
     }
 
-    @GetMapping(value = "/adminsGroupings")
-    public ResponseEntity<String> adminsGroupings() {
-        logger.info("Entered REST adminsGroupings...");
-        String currentUsername = policy.sanitize(userContextService.getCurrentUsername());
-        String uri = API_2_1_BASE + "/admins-and-groupings";
-        return httpRequestService.makeApiRequest(currentUsername, uri, HttpMethod.GET);
+    @GetMapping(value = "/groupingAdmins")
+    public ResponseEntity<String> groupingAdmins(Principal principal) {
+        logger.info("Entered REST groupingAdmins...");
+        String principalName = policy.sanitize(principal.getName());
```

Comment:
Make sure to use `String currentUsername = policy.sanitize(userContextService.getCurrentUsername());` instead of `String principalName = policy.sanitize(principal.getName());`. There was a recent change to remove the Principal parameter. Thanks!